README.md

```diff
@@ -27,6 +28,33 @@ The SDK is a Kotlin library for sending user properties and events to the Appcue
 
 ## 🚀 Getting Started
 
+### Prerequisites
+
+Your application's `build.gradle` must have a `compileSdkVersion` of 34+ and `minSdkVersion` of 21+
+
+```
+android {
+    compileSdkVersion 34
+
+    defaultConfig {
+        minSdkVersion 21
+    }
+}
+```
+
+In addition because of Compose 1.6 update (internally used by the SDK), it is required to either:
+
+1. apply `kotlin-android` plugin in app's build.gradle file.
+   ```
+   plugins {  
+     id 'com.android.application' 
+     id 'kotlin-android' 
+   }
+   ```
+2. **OR** Update Android Gradle Plugin 8.4.0+
+
+> [Related ticket](https://issuetracker.google.com/issues/328687152) regarding this later found issue when updating Compose dependency from 1.5 to 1.6+
```

Comment:
can say

[Related Google issue](https://issuetracker.google.com/issues/328687152) regarding usage of the Jetpack Compose dependency versions 1.6+

Reply:
Thanks!